### DISCUSSION TITLE
mpox incubation period

### DISCUSSION THREAD

**Discussion by sbfnk** [epinowcast/epidist#411] [Ideas]
I was curious about what the estimates of [Marziano et al.](https://doi.org/10.1101/2024.05.10.24307157) would look like when using this package and accounting for censoring.

See https://github.com/sbfnk/epidist/blob/mpox/vignettes/mpox.md

It turns out a different distribution (gamma vs. weibull) is preferred but only very slightly (using a different information criterion), and the estimated mean is slightly different. Overall in this case it seems like it might be fine to use the uncorrect method. Obviously this could all be explored in more detail, and/or the same applied to the serial interval estimate done in the paper.

**Comment by seabbs** [comment]
This is great. It would be nice to update this to use the latest version of epidist and to use the latent model vs the uniform censoring window approximation and see if that makes a difference. 

When doing that it would also be neat to include a comparison in vignette of the naive vs adjusted estimates and as a stretch goal simulation some observation scenarios to assess the impact there. 

Definitely seems worth adding this to the package imo.

**Reply by seabbs** [reply]
Just to follow up on this I haven't read the original paper but assuming they just treated it as continuous I would expect a difference in mean here of only 5-10% with probably slightly more impact in the tails. I think quantifying that bias more clearly (using the theory and some simulations is a good thing to add to somewhere like the primary censored paper)
